types/relay-runtime/lib/store/RelayModernStore.d.ts

```diff
@@ -22,7 +21,11 @@ import {
 } from './RelayConnection';
 
 export class RelayModernStore implements Store {
-    constructor(source: MutableRecordSource, gcScheduler?: Scheduler, operationLoader?: OperationLoader | null);
+    constructor(source: MutableRecordSource, options?: {
+        gcScheduler?: Scheduler | null,
```

Comment:
`gcScheduler?: ?Scheduler` in Flow means `gcScheduler?: Scheduler | null | undefined` more accurately, I think.